tensorflow_quantum/datasets/spin_system.py

```diff
@@ -312,6 +312,248 @@ def tfi_chain(qubits, boundary_condition="closed", data_dir=None):
             labels.append(1)
         else:
             labels.append(2)
+    # Make sure that the data is ordered from g=0.2 to g=1.8.
+    _, resolved_circuits, labels, hamiltonians, additional_info = zip(*sorted(
+        zip(order_parameters, resolved_circuits, labels, hamiltonians,
+            additional_info)))
+
+    return resolved_circuits, labels, hamiltonians, additional_info
```

Comment:
Was there a change made to `tfi_chain` os this the diff acting weird ?

Reply:
No changes here. Don't know why this shows up.